stablehlo/dialect/StablehloOps.td

```diff
@@ -2627,6 +2640,12 @@ def StableHLO_ReducePrecisionOp :
     implementations.
 
     See https://www.tensorflow.org/xla/operation_semantics#reduceprecision.
+
+    Example:
+
+    ```mlir
+    %0 = mhlo.reduce_precision %arg0, format = e8m10 : tensor<3x4xf32>
```

Comment:
Looks like PR description should be updated to use the new assemblyFormat?

Reply:
PR description has `%0 = stablehlo.reduce_precision %arg0, format = e8m10 : tensor<3x4xf32>`.

That said, this line says `mhlo` which is incorrect. Fixed all uses of mhlo in doc examples.